quiz/utils.py

```diff
@@ -0,0 +1,34 @@
+
+from django.db.models.fields import SlugField
+from django.shortcuts import get_object_or_404
+from organization.models import InternalGroup
+from users.models import User
+from quiz.models import Participant, Quiz
+
+
+def user_quiz_pool_helper(quiz, internal_group):
+    pool = None
+    group = None
+    if internal_group == "new-members":
+        category = InternalGroup.objects.filter(
+            type=InternalGroup.Type.INTERNAL_GROUP
+        ).order_by("?")
+    else:
+        category = InternalGroup.objects.filter(slug=internal_group).order_by("?")
+    pool = []
+    for group in category:
+        pool.extend([membership.user for membership in group.active_members])
+    return pool
+
+    pool_setup = user_quiz_pool_helper(quiz, internal_group)
+    quiz.fake_users.set(pool_setup)
+
+def guess_helper(quiz, user_id):
+    participant = quiz.current_guess
+    participant.guessed_user = User.objects.get(pk=user_id)
```

Comment:
Maybe use a bit more descriptive name?

```suggestion
def make_a_guess(quiz, guess_user_id):
    participant = quiz.current_guess
    participant.guessed_user = User.objects.get(pk=guess_user_id)
```

Reply:
![bilde](https://user-images.githubusercontent.com/31571115/147354690-512983fe-b0fd-48ee-ab22-436f223b52ff.png)